### PR TITLE
Bump serde-byte-array and change backend directory

### PR DESCRIPTION
**Pull request by robin-nitrokey**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ description = "Authentication extension and backend for Trussed"
 
 [dependencies]
 serde = { version = "1", default-features = false }
-serde-byte-array = "0.1.0"
+serde-byte-array = "0.1.1"
```
Comment (sosthene-nitrokey) [inline Cargo.toml:15]: Why not `0.1.2`? That way we have the `Copy` impl.

**Review by sosthene-nitrokey — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (robin-nitrokey) [reply Cargo.toml:15]: Do we need it in this crate?

Comment (sosthene-nitrokey) [reply Cargo.toml:15]: We need it in #10

Comment (robin-nitrokey) [reply Cargo.toml]: I see.  I’ve bumped it to 0.1.2.

**Pull request merged**